.gitlab-ci.yml

```diff
@@ -37,12 +38,17 @@ variables:
   DEVICEMONITOR_REV: "master"
   MONITOR_CLIENT_REV: "master"
   REPORTING_REV: "master"
-  MENDER_GATEWAY_REV: "master"
 
   # Versions of independent components
   MENDER_BINARY_DELTA_VERSION: "latest"
   MENDER_CONFIGURE_MODULE_VERSION: "latest"
 
+  # The revision variables for the following components need to be defined also
+  # in the project's CI/CD settings because they are used in include statements
+  # See comment here: https://gitlab.com/gitlab-org/gitlab/-/issues/219065#note_619008094
+  # Defined here: https://gitlab.com/Northern.tech/Mender/mender-qa/-/settings/ci_cd
+  MENDER_GATEWAY_REV: "master"
+
```

Comment:
ok either I am crazy, or we have just moved it 2 lines down?

Reply:
We have moved it and documented that it needs to be set into CI/CD settings.